src/Powercord/coremods/moduleManager/commands/share.js

```diff
@@ -41,42 +34,31 @@ function autocompleteFlags (args) {
 
 module.exports = {
   command: 'share',
-  description: 'Share a plugin or theme that you have',
+  description: Messages.REPLUGGED_COMMAND_SHARE_DESC,
   usage: '{c} [ plugin/theme ID]',
   executor ([ id, ...args ]) {
     const isPlugin = args.includes('--plugin') || powercord.pluginManager.plugins.has(id);
     if (!args.includes('--theme') && !args.includes('--plugin')) {
       const isTheme = powercord.styleManager.themes.has(id);
 
       if (!isPlugin && !isTheme) { // No match
-        return resp(false, `Could not find plugin or theme matching "${id}".`);
+        return resp(false, Messages.REPLUGGED_ENTITY_NOT_FOUND.replace('{id}', id);
       } else if (isPlugin && isTheme) { // Duplicate name
-        return resp(false, `"${id}" is in use by both a plugin and theme. You will have to specify with --theme or --plugin flag.`);
+        return resp(false, Messages.REPLUGGED_COMMAND_SHARE_BOTH_ENTITY.replace('{id}', id);
       }
     }
     
     const manager = args.includes('--plugin') || isPlugin ? powercord.pluginManager : powercord.styleManager;
     const entity = manager.get(id);
+    const url = getURL(entity);
     
-    let data = fs.readFileSync(path.resolve(entity.entityPath, '.git', 'config'), 'utf8');
-    data = data.split('\n').map(e => e.trim());
-    
-    let url = '';
-    for (let i = 0; i < data.length; i++) {
-      if (data[i].startsWith('url = ')) {
-        url = formatUrl(data[i]);
-        break;
-      }
-    }
     if (url !== '') {
       if (args.includes('--open')) {
         openURL(url);
         return;
       }
-      if ((!args.includes('--no-repo')) || args.includes('--repo')) {
-        if ((args.includes('--no-embed')) && !args.includes('--embed')) {
-          url = `<${url}>`;
-        }
+      if (args.includes('--repo')) {
+        url = `<${url}>`;
       } else {
         url = `<https://replugged.dev/install?url=${url}>`;
       }
```

Comment:
re-implement `--embed` flag
```suggestion
      if (!args.includes('--repo')) {
        url = `https://replugged.dev/install?url=${url}`;
      }
      if (!args.includes('--embed')) {
        url = `<${url}>`;
      }
```